modules/admin_manual/pages/configuration/files/previews_configuration.adoc

```diff
@@ -84,7 +84,8 @@ and change:
 rights="none" --> rights="read|write"
 ----
 
-After changing the policy file for ImageMagic, restart your Apache web server or your php-fpm service.
+After changing the policy file for ImageMagick, restart your Apache web server or your php-fpm service.
+ImageMagick uses ghostscript for converting PDF. Please also make sure, that ghostscript is installed and PDF format is listed in e.g. `convert -list format`.
```

Comment:
> ...PDF format is listed in e.g. `convert -list format`.

Where can that be found? Is it a config file? Which SW?

Reply:
Convert is the main shell command for imagemagick. Silly name, but that's what it is.
We require `apt install imagemagick` anyway, after that you have `convert`.

Does that answer your Zwetschgn?

Reply:
🤦‍♂️ 
ok, I will do a small text change for clarification...

Reply:
fixed